modified home pages, especially graphs at bottom

malawi-mike opened the pull request.

jdepree commented on project.php:
```diff
@@ -153,6 +153,7 @@ class="btn-large grey">
 
 		</div>
 	</div>
+	</div>
```
do you really want this?  I might delete it.

jdepree replied:
hmmm... I thought it would fix something to remove it, but it didn't.  Did you have a reason for adding this?

jdepree replied:
ha, removing it did fix it but it was caching.  Maybe I can put in some sort of debug flag on that cache command...

malawi-mike replied:
which button was this? i'm confused.

jdepree commented:
the Control Villages color was painful to look at, so I replaced it.  I'm not crazy about the new one, so we might try again

malawi-mike commented:
totally agree. too faint. the color you selected works for me. probably
better than black.

On Thu, Oct 26, 2017 at 10:19 AM, Jeff DePree <notifications@github.com>
wrote:

> the Control Villages color was painful to look at, so I replaced it. I'm
> not crazy about the new one, so we might try again
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Village-X-Org/Site-v2/pull/87#issuecomment-339681605>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AcyICrFaofKN8xXf19U4ZTPCZz0aP6FRks5swJUJgaJpZM4QGflJ>
> .
>



-- 
Mike Buckler, CEO & General Counsel
Village X Org <http://www.villagexapp.com>
mike@villagex.org
202-360-9931
<http://www.villagexapp.com>

malawi-mike commented:
could you make the same change in the graphs on the project and impacts
pages? also, the Data Tracker info on the homepage is intended to be
updated automatically from the projects spreadsheet (although for now it
doesn't seem necessary).

spent this morning working on recognizing donors on project pages. here's
what i've got. currently have toasts for names but watsi uses tooltips,
which i like. however, it cannot get Materialize tooltips to work. seems to
be a conflict somewhere. have you had any luck with them?

will soon turn my attention back to blogs. probably best to use an outside
service like medium, with a custom url. seems like coding all the blog
features we want (e.g., keywords, search) would take time. not sure what
watsi <http://blog.watsi.org/> does. any way to find out?


​

On Thu, Oct 26, 2017 at 2:34 PM, Michael Buckler at Village X Org <
mike@villagex.org> wrote:

> totally agree. too faint. the color you selected works for me. probably
> better than black.
>
> On Thu, Oct 26, 2017 at 10:19 AM, Jeff DePree <notifications@github.com>
> wrote:
>
>> the Control Villages color was painful to look at, so I replaced it. I'm
>> not crazy about the new one, so we might try again
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/Village-X-Org/Site-v2/pull/87#issuecomment-339681605>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/AcyICrFaofKN8xXf19U4ZTPCZz0aP6FRks5swJUJgaJpZM4QGflJ>
>> .
>>
>
>
>
> --
> Mike Buckler, CEO & General Counsel
> Village X Org <http://www.villagexapp.com>
> mike@villagex.org
> 202-360-9931 <(202)%20360-9931>
> <http://www.villagexapp.com>
>



-- 
Mike Buckler, CEO & General Counsel
Village X Org <http://www.villagexapp.com>
mike@villagex.org
202-360-9931
<http://www.villagexapp.com>
